mtags/src/main/scala/scala/meta/internal/mtags/CoursierComplete.scala

```diff
@@ -0,0 +1,81 @@
+package scala.meta.internal.mtags
+
+import scala.collection.JavaConverters._
+import scala.concurrent.Await
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+import scala.concurrent.duration._
+import scala.util.matching.Regex
+
+import scala.meta.internal.tokenizers.Chars
+
+import coursierapi.Complete
+
+object CoursierComplete {
```

Comment:
Can we use this also for AmmoniteCompletions?

Reply:
Of course, but I thought about doing this in a follow-up, together with Ammonite completions for scala 3.

Reply:
Ok sure, we can do it next